addon/props/model_props.py

```diff
@@ -86,6 +86,21 @@ class SOURCEOPS_ModelProps(bpy.types.PropertyGroup):
         default=False,
     )
 
+    custom_transform_source: bpy.props.EnumProperty(
+        name='Transform Source',
+        description='Method of specifying $origin and $scale\nEither manually specified in this Panel, or via an object',
+        items=[
+            ('MANUAL', 'Manual Input', 'Specify the Transform manually in this panel', 'NONE', 1),
+            ('CUSTOM_OBJECT', 'Object', 'Use an Object\'s transforms\nIf it is None, then falls back to Manual Input', 'NONE', 2)
```

Comment:
You can get rid of the last two items in each tuple, they're only needed if you use icons.
Oh and I'd change `CUSTOM_OBJECT` to just `OBJECT`, because uh it could hardly *not* be custom.